file-check/action.yaml

```
@@ -0,0 +1,54 @@
+name: file-check
+description: ""
+
+inputs:
+  files:
+    description: ""
+    required: true
+  condition:
+    description: ""
+    required: true
+outputs:
+  exists:
+    description: ""
+    value: ${{ steps.file-check.outputs.exists }}
+
+runs:
+  using: composite
+  steps:
+    - name: File check
+      id: file-check
+      run: |
+        set -e
+
+        # Split files into an array
+        input_files="${{ inputs.files }}"
+        file_array=(${input_files})
+
+        # Loop through files
+        exist_files=()
+        for FILE in "${file_array[@]}"; do
+          if [[ -f $FILE ]]; then
+            echo "File $FILE exists"
```

```suggestion
            echo "File exists: $FILE"
```

Or this looks nice to me.
```
            echo "File '$FILE' exists."
```